content/post/0-to-60-analytics-stack.md

```diff
@@ -0,0 +1,79 @@
+
+I made one specific mistake here that I want to share: Looker was much more expensive than the other options, and, to explain the cost, I leaned on the value proposition of teams not needing to hire a dedicated analyst to write SQL queries. This was a mistake for two reasons:
+
+1.  Business users still need a baseline skill level to use Looker. A surprisingly effective rule-of-thumb is that if a person is comfortable with pivot tables in excel, they can do Looker. Do a quick calculation of how many people in your org need data, and of those, how many of them you would trust to make a pivot table.
+
+2. There are benefits to hiring an analyst beyond simply doing SQL queries. Looker saved us from hiring an analyst, but that analyst could also have added value by connecting the dots of cross-functional business requests into product priorities. Unless you are severely budget constrained, it doesn’t hurt to have another bright technical person around, and the earlier you hire them, the better.
+
+Here is a simple chart I wish I would have used to diagnose analytics needs:
+
+![Analyst or Self Service](https://nelsonauner.com/images/analysts_or_self_serve.png)
+
+
+This concludes the three main “courses” of an analytics stack. If you are still hungry, I will touch briefly on two increasingly necessary components as you continue to grow:
+
+## Business Logic Layer:  
+- You might consider: Looker, DBT
+
+ Once you get to more than 3 SQL writers, each will use slightly different JOIN conditions and aggregations, and it will cause problems. These problems may be technical (the same expensive queries getting run repeatedly) but certainly analytical problems as different teams build out slightly different queries for the same metrics.
+Business logic layers solve this problem by centralizing and documenting the (business logic <-> data) interface.
+
+I was comparing DBT with Looker, but the important takeaway is to consider the tradeoffs between a comprehensive system  DBT overlaps a lot with LookML in this regard. If you don’t use Looker, you should definitely use DBT. If you do use Looker, the gains to using DBT are less. I opted to use DBT because it will make it much easier to move off of Looker, if I ever need to, allows us to use DBT’s cleaned data for data science instead of trapping that data in Looker, and has some more advanced features not found in LookML's PDTs. You can opt for DBT open source or their paid hosted version, DBT Cloud. DBT Cloud is more user friendly, but still requires a fair amount of technical knowledge to operate.
```

Comment:
I'm a bit hesitant to use words like "definitely". I'd suggest tempering / qualifying the language here.